<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
	background-color: rgb(0, 0, 0);
}


.QPushButton{
	background-color: rgb(66, 66, 66);
	border-radius:39%;
	margin: 6px;
	font-size: 37px;
	font-family: Lucida Sans; 
	color: rgb(255,255,255);
	text-align: center;
}

#btnLimpar, #btnPosNeg, #btnPorcentagem{
	background-color: rgb(176, 176, 176);
	color: rgb(0, 0, 0);
	text-align: center;
	
}

#btnDivisao, #btnMultiplicacao, #btnSubtracao, #btnSoma, #btnTotal{
	background-color: rgb(255,140,0);
	color: rgb(255,255,255);
	text-align: center;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLimpar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPosNeg">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPorcentagem">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMultiplicacao">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>ร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDivisao">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVirgula">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTotal">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>490</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
    <property name="shortcut">
     <string>Enter</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSoma">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
    <property name="shortcut">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSubtracao">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: left;
</string>
    </property>
    <property name="text">
     <string>  0</string>
    </property>
   </widget>
   <widget class="QLabel" name="btnTela">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
